src/bin/index.js

```diff
@@ -94,6 +102,120 @@ yargs(hideBin(process.argv))
+  .command(
+    ['lightnet <sub-command> [options]'],
+    'Manage the lightweight Mina blockchain for zkApps development and testing purposes.\nYou can find more information about the Docker image in use at\nhttps://hub.docker.com/r/o1labs/mina-local-network',
+    (yargs) => {
+      yargs
+        .command(
+          [
+            'start [mode] [type] [proof-level] [mina-branch] [archive] [sync] [pull]',
+          ],
+          'Start the lightweight Mina blockchain network Docker container.',
+          {
+            mode: {
+              alias: 'm',
+              demand: false,
+              string: true,
+              hidden: false,
+              choices: Constants.lightnetModes,
+              default: 'single-node',
+              description:
+                'Whether to form the network with one node or with multiple network participants.\n"single-node" value will make the network faster.',
+            },
+            type: {
+              alias: 't',
+              demand: false,
+              string: true,
+              hidden: false,
+              choices: Constants.lightnetTypes,
+              default: 'fast',
+              description:
+                'Whether to configure the network to be fast or slower with closer-to-real-world properties.',
+            },
+            'proof-level': {
+              alias: 'p',
+              demand: false,
+              string: true,
+              hidden: false,
+              choices: Constants.lightnetProofLevels,
+              default: 'none',
+              description: '"none" value will make the network faster.',
+            },
+            'mina-branch': {
+              alias: 'b',
+              demand: false,
+              string: true,
+              hidden: false,
+              choices: Constants.lightnetMinaBranches,
+              default: 'rampup',
+              description:
+                'One of the major Mina repository branches the artifacts were compiled against.',
+            },
+            archive: {
+              alias: 'a',
+              demand: false,
+              boolean: true,
+              hidden: false,
+              default: true,
+              description:
+                'Whether to start the Mina Archive process and Archive Node API application within the Docker container.',
+            },
+            sync: {
+              alias: 's',
+              demand: false,
+              boolean: true,
+              hidden: false,
+              default: true,
+              description:
+                'Whether to wait for the network to reach the synchronized state.',
+            },
+            pull: {
+              alias: 'u',
+              demand: false,
+              boolean: true,
+              hidden: false,
+              default: true,
+              description:
+                'Whether to pull the latest version of the Docker image from the Docker Hub.',
```

Comment:
@shimkiv  advantages and disadvantages? default is true, so when would we want to use false?

Reply:
Docker images are updated each night and pulling the new image will takes resources and time should you start the network next day from scratch. It is useful to set `false` when you develop/test zkapp functionality using same version of o1js and Mina dependencies. The reason it is not set to `false` by default is because I was thinking to encourage people always use "latest and greatest" but not sure already.